assets.mjs

```diff
@@ -0,0 +1,44 @@
+import sharp from "npm:sharp@0.33.5";
+
+// Names of the SVGs to use for generation
+const logoFileNames = ["css", "css.square", "css.dark", "css.light"];
+
+// Start the generation process for each logo
+for (const logoFileName of logoFileNames) {
+  // Extract the type of the logo, when the type is not provided it defaults to "primary"
+  const [_, logoType = "primary"] = logoFileName.split(".");
+
+  // Load the SVG file into sharp
+  const image = await sharp(`${logoFileName}.svg`);
+
+  // Specify and prepare the output folder
+  const outputFolder = `./${logoType}`;
+
+  // Remove the output folder if it exists
+  try {
+    await Deno.remove(outputFolder, { recursive: true });
+  } catch (error) {
+    if (!(error instanceof Deno.errors.NotFound)) {
+      throw error;
+    }
```

Comment:
Is this step necessary?

Reply:
Deno will crash if the folder does not exist, it's just error handling in case they don't.
Better safe than sorry

Reply:
I meant to ask whether removing the folder at all was necessary 😆